akamai/resource_akamai_property_activation.go

```diff
@@ -366,18 +390,31 @@ func findExistingActivation(property *papi.Property, activation *papi.Activation
 		papi.StatusZone3:               true,
 	}
 	for _, a := range activations.Activations.Items {
+		edge.PrintfCorrelation("[DEBUG]", correlationid, fmt.Sprintf(" SET VERSION UPDATE LOOP TEST loop values %s version %d Activationpassed in  %s version %d  \n", a.Network, a.PropertyVersion, activation.Network, activation.PropertyVersion))
```

Comment:
Did we need to log the full activation range here for context?  Was that why we have a separate loop so you can tell when there were conflicting/overlapping activations?

Reply:
Yes just wanted to see loop contents prior as secondary loop exited prematurely will remove